fcrepo-http-api/src/main/java/org/fcrepo/http/api/responses/StreamingBaseHtmlProvider.java

```diff
@@ -262,50 +235,41 @@ protected Context getContext(final Model model, final Node subject) {
 
     private Template getTemplate(final Model rdf, final Node subject,
                                  final Annotation[] annotations) {
```

Comment:
No idea -- it should be.

I tried not to modify too many existing fn signatures, but since this is `private` there's no reason not to